src/data/matrix/basic.lean

```diff
@@ -138,6 +138,10 @@ lemma map_smul [has_scalar R α] [has_scalar R β] (f : α → β) (r : R)
   (hf : ∀ a, f (r • a) = r • f a) (M : matrix m n α) : (r • M).map f = r • (M.map f) :=
 ext $ λ _ _, hf _
 
+lemma is_smul_regular [has_mul α] (k : α) (hk : is_left_regular k) :
```

Comment:
```suggestion
lemma _root_.is_smul_regular.matrix [has_mul α] (k : α) (hk : is_left_regular k) :
```

Reply:
I think this statement should be about `is_smul_regular α k`?